packages/@coorpacademy-components/src/molecule/dashboard/cards-list/index.js

```diff
@@ -155,18 +142,23 @@ class CardsList extends React.Component {
     return getOr(0, 'offsetWidth', this.cardsWrapper);
   }
 
-  cardsWidth() {
-    return sumBy('scrollWidth', this.cards);
+  // eslint-disable-next-line class-methods-use-this
+  getScrollWidth(b) {
+    return getOr(0, 'scrollWidth', b);
+  }
+
+  getPossiblePositions() {
+    return this.cards.map((card, index, arr) => {
+      return arr.slice(0, index).reduce((a, b) => {
+        return a + this.getScrollWidth(b);
+      }, 0);
+    });
   }
 
-  getPossiblePositions(filter) {
-    return this.cards
-      .map((card, index, arr) => {
-        return arr.slice(0, index).reduce((a, b) => {
-          return a + getOr(0, 'scrollWidth', b);
-        }, 0);
-      })
-      .filter(filter);
+  getPossiblePages() {
+    return this.getPossiblePositions().map((el, index) =>
```

Comment:
`el` comme nom de variable pour l'itérateur est pas très parlant.
De base j'aurais dit _élements_ mais d'après le corps de la fonction c'est plus une taille vu qu'elle est dans un Math.ceil additionné à une width.

is it `elementWidth`?

Reply:
ah oui , c'est la position de l’élément